examples/fluent-bit/oomkill-prevention/firelens-full-filesystem-example/fluent-bit-image/extra.conf

```diff
@@ -49,4 +49,7 @@
     auto_create_group true
     retry_limit 2
     # limit the filesystem storage that this output can use
-    storage.total_limit_size 1G
\ No newline at end of file
+    storage.total_limit_size 1G
+    # from aws-for-fluent-bit v2.32.0 and on, to support higher throughput logging,
+    # set workers to a high value such as 5 or the number of cores on your host
```

Comment:
Can you please replace all occurrences of "high value" with a different phrasing?

Reply:
Changed wording to recommended wording